aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
```

Comment:
Unfortunately ... pardon the double-negative ... I **_couldn't_** convince them to **_NOT_** close void elements. :cry:

Reply:
There's been a long-standing situation with Razor that you probably already know about. *Razor needs closed tags.* In these sample app cases it's ok to not close them, but I've been forced at gunpoint :boy: :gun: by Razor in professional apps to close some things that I didn't want to close. Anyway ... the spec is ok with it. It just feels funky in a way. The Razor thing might be the reason that they want to close *everything*, and I totally understand the value in that vis-a-vis Razor. Closing everything takes the guesswork out of it.